x-pack/legacy/plugins/siem/cypress/integration/lib/timeline/selectors.ts

```diff
@@ -36,3 +36,7 @@ export const TOGGLE_TIMELINE_EXPAND_EVENT = '[data-test-subj="expand-event"]';
 
 /** The body of the timeline flyout */
 export const TIMELINE_FLYOUT_BODY = '[data-test-subj="eui-flyout-body"]';
+
+export const CLOSE_TIMELINE_BTN = '[data-test-subj="close-timeline"]';
+
+export const CLOSE_PROVIDER_BADGE_BTN = '[data-test-subj="closeProviderBadge"]';
```

Comment:
To verify the `clearTimeline()` helper is working, I added an invocation of `clearTimeline()` to the end of  the `it('renders the data provider of a host dragged from the All Hosts widget on the hosts page`) test in `x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/timeline/data_providers.spec.ts`. I observed that the data provider was not being cleared by `clearTimeline()`, and traced the issue to this selector, which doesn't appear to match anything.

Changing the selector to the following fixed it:

```
export const CLOSE_PROVIDER_BADGE_BTN = 'button[title="Remove Data Provider"]';
```